rspack.config.ts

```diff
@@ -279,93 +301,55 @@ const appConfig: Configuration = {
       },
       {
         test: /\.(woff|woff2|ttf|eot|svg|png|gif|ico|jpg|mp4)($|\?)/,
         type: 'asset',
       },
     ],
-    noParse: [
-      // don't parse known, pre-built javascript files (improves webpack perf)
-      /jed\/jed\.js/,
-      /marked\/lib\/marked\.js/,
-      /terser\/dist\/bundle\.min\.js/,
-    ],
   },
   plugins: [
     /**
      * Adds build time measurement instrumentation, which will be reported back
      * to sentry
      */
-    new SentryInstrumentation(),
-
-    // Do not bundle moment's locale files as we will lazy load them using
-    // dynamic imports in the application code
-    new webpack.IgnorePlugin({
-      contextRegExp: /moment$/,
-      resourceRegExp: /^\.\/locale$/,
-    }),
+    // new SentryInstrumentation(),
 
     /**
      * TODO(epurkhiser): Figure out if we still need these
      */
-    new webpack.ProvidePlugin({
+    new rspack.ProvidePlugin({
       process: 'process/browser',
       Buffer: ['buffer', 'Buffer'],
     }),
 
-    /**
-     * Extract CSS into separate files.
-     */
-    new MiniCssExtractPlugin({
-      // We want the sentry css file to be unversioned for frontend-only deploys
-      // We will cache using `Cache-Control` headers
-      filename: 'entrypoints/[name].css',
-    }),
-
     /**
      * Defines environment specific flags.
      */
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify(env.NODE_ENV),
-        IS_ACCEPTANCE_TEST: JSON.stringify(IS_ACCEPTANCE_TEST),
-        DEPLOY_PREVIEW_CONFIG: JSON.stringify(DEPLOY_PREVIEW_CONFIG),
-        EXPERIMENTAL_SPA: JSON.stringify(SENTRY_EXPERIMENTAL_SPA),
-        SPA_DSN: JSON.stringify(SENTRY_SPA_DSN),
-        SENTRY_RELEASE_VERSION: JSON.stringify(SENTRY_RELEASE_VERSION),
-      },
+    new rspack.DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify(env.NODE_ENV),
+      'process.env.IS_ACCEPTANCE_TEST': JSON.stringify(env.IS_ACCEPTANCE_TEST),
+      'process.env.DEPLOY_PREVIEW_CONFIG': JSON.stringify(env.DEPLOY_PREVIEW_CONFIG),
+      'process.env.EXPERIMENTAL_SPA': JSON.stringify(env.EXPERIMENTAL_SPA),
+      'process.env.SPA_DSN': JSON.stringify(env.SPA_DSN),
+      'process.env.SENTRY_RELEASE_VERSION': JSON.stringify(env.SENTRY_RELEASE_VERSION),
     }),
 
-    /**
-     * This removes empty js files for style only entries (e.g. sentry.less)
-     */
-    new FixStyleOnlyEntriesPlugin({verbose: false}),
-
-    ...(SHOULD_FORK_TS
-      ? [
-          new ForkTsCheckerWebpackPlugin({
-            typescript: {
-              configFile: path.resolve(__dirname, './config/tsconfig.build.json'),
-              configOverwrite: {
-                compilerOptions: {incremental: true},
-              },
-            },
-            devServer: false,
-            // memorylimit is configured in package.json
-          }),
-        ]
-      : []),
+    // ...(SHOULD_FORK_TS
+    //   ? [
+    //       new ForkTsCheckerWebpackPlugin({
```

Comment:
Fork TS is very slow, will be the bottleneck of rspack build speeds. If you can run tsc as a separate command, it will keep the builds very fast.